Bash_scripts/HW28/Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+pipeline {
+  agent any
+    parameters {
+    string(name: 'PATHTO', defaultValue: 'Bash_scripts/HW27', description: null)
+    string(name: 'VERSION', defaultValue: 'v1', description: null)
+    string(name: 'IMG_NAME', defaultValue: 'hw28', description: null)
+    string(name: 'USR_REPO', defaultValue: 'engelko', description: null)
+  }
+  stages {
+    stage('Shell script 0') {
+      steps {
+        sh '''
+        #!/bin/bash
+        cd $PATHTO
+        SHELL_SCRIPTS=$(find . -name \'*.sh\')
+        for var in $SHELL_SCRIPTS ; do
+        shellcheck "$var"
+        done
+        '''
+      }
+    }    
+    stage('Shell script 1') {
+      steps {
+        sh '''
+        #!/bin/bash
+        DOCKERFILE=$(find "$PATHTO" -name \'Dockerfile\')
```

Comment:
Good practise to set full path to file as a variable. In this case if you want to change file name you need do to it only in one place.

Reply:
fixed